modules/auxiliary/scanner/http/wordpress_xmlrpc_massive_bruteforce.rb

```diff
@@ -0,0 +1,171 @@
+##
+# This module requires Metasploit: http://www.metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HTTP::Wordpress
+
+  def initialize(info = {})
+    super(update_info(
+              info,
+              'Name'            => 'WordPress XMLRPC Massive Bruteforce ',
+              'Description'     => %q{Wordpress Massive Burteforce attacks via wordpress XMLRPC service.},
+              'License'         => MSF_LICENSE,
+              'Author'          =>
+                  [
+                      'William (WCoppola@Lares.com)',
+                      'Sabri (@KINGSABRI)'
+                  ],
+              'References'      =>
```

Comment:
Indentation is out. Please use two space soft indentation as described at https://github.com/rapid7/metasploit-framework/wiki/Indentation-Standards